docs/docs/aleph_zero/sdk.md

```diff
@@ -185,56 +182,38 @@ await psp22.approve(account, invariant.contract.address.toString(), tokenXAmount
 await psp22.setContractAddress(poolKey.tokenY)
 await psp22.approve(account, invariant.contract.address.toString(), tokenYAmount)
 
-// open up position
+// create position
 const createPositionResult = await invariant.createPosition(
   account,
   poolKey,
-  lowerTick,
-  upperTick,
+  lowerTickIndex,
+  upperTickIndex,
   positionLiquidity,
   initSqrtPrice,
   0n
 )
-console.log(createPositionResult.hash) // print transaction hash
+console.log(createPositionResult.hash)
 ```
 
-### Perform swap
+:::tip Output
+7999999999880n 8000000000000n <br/>
+0x652108bb36032bc386fec2eef3f483f29970db7bdbdc9a1a340e279abd626ee2
+:::
+
+### Swap tokens
 
 :::info How to calculate input amount
 In order to calculate input amount, we have to multiply actual token amount you want to swap times 10 to the power of decimal.
```

Comment:
The box regarding how to express the token amount should be present at the first usage, so in position creation, and not repeated later